config/config.go

```diff
@@ -163,16 +164,23 @@ func buildBpfFilter() string {
 	// reference links:
 	// https://www.middlewareinventory.com/blog/tcpdump-capture-http-get-post-requests-apache-weblogic-websphere/
 	// https://www.middlewareinventory.com/ascii-table/
+	// tcp[((tcp[12:1] & 0xf0) >> 2):<num> means skip the ip & tcp headers, then get the next <num> bytes and match hex
+	// bpf insists that we must use 1, 2, or 4 bytes
+	matchFirstFourBytesTo := "tcp[((tcp[12:1] & 0xf0) >> 2):4] = 0x"
+
```

Comment:
I think this could be simplified by looping over an array of method strings.

Something like:
```golang
methods := []string{"GET", ...}
filters := []string{}
for _, method := range methods {
    filters = append(filters, matchFirstFourBytesTo + hex.EncodeToString([]byte(method))
}
return strings.Join(filters, " or ")